core/rovecomm.py

```diff
@@ -22,8 +22,10 @@
     3: 'H',
     4: 'l',
     5: 'L',
-    6: 'q',  # int64, this needs to stay here to not break RED. Leave this comment here Skelton.
-    7: 'd',  # double
+    6: 'f',
+    7: 'q',  # int64, this needs to stay here to not break RED. Leave this comment here Skelton.
+    8: 'd',  # double
+    9: 's',
```

Comment:
Think about changing this to "c" to make it a little more transparent this is a character array